src/index.js

```diff
@@ -24,14 +24,20 @@ if (!config.rancher.url) {
 	console.error("RANCHER_URL not set");
 }
 
-if (config.slack.channels.length === 0) {
+let channels = null;
+if (config.slack.channels !== null && config.slack.channels !== "") {
+	channels = config.slack.channels.split(",");
```

Comment:
The code should trim the tokens to prevent situations where the slack channels env var has whitespaces (or tabs, etc) between tokens

ex: ```SLACK_TOKENS=chan1,    chan2```

additionally, there should be some tests around the parsing